doc/reference/storage_powerflex.md

```diff
@@ -0,0 +1,151 @@
+
+This driver also behaves differently than other drivers in that it provides remote storage.
+As a result and depending on the internal network, storage access might be a bit slower than for local storage.
+On the other hand, using remote storage has big advantages in a cluster setup, because all cluster members have access to the same storage pools with the exact same contents, without the need to synchronize storage pools.
+
+When creating a new storage pool using the `powerflex` driver, LXD tries to discover one of the SDT from the given storage pool.
+This SDT can be overwritten using [`powerflex.sdt`](storage-powerflex-pool-config).
+LXD instructs the NVMe initiator to connect to all the other SDT when first connecting to the subsystem.
+
+Due to the way copy-on-write works in PowerFlex, snapshots of any volume don't rely on its parent.
+As a result, volume snapshots are fully functional volumes themselves from whom additional snapshots can be taken.
+This tree of dependencies is called the PowerFlex vTree.
+Both volumes and their snapshots get added as standalone NVMe disks to the LXD host.
+
+(storage-powerflex-volume-names)=
+### Volume names
+
+Due to a [limitation](storage-powerflex-limitations) in PowerFlex, volume names cannot exceed 31 characters.
+In order for LXD to identify the volumes type and snapshots, special identifiers are appended to the volumes name:
+
+Type            | Identifier   | Example
+:--             | :---         | :----------
+Container       | `c_`         | `c(_project1)_c1`
+Virtual machine | `v_`         | `v(_project2)_v1`
+Image           | `i_`         | `i(_project3)_img`
+Custom volume   | `u_`         | `u(_project4)_vol`
+
+When creating volumes in the `default` project its identifier gets omitted.
+This does not apply for custom storage volumes since their name may contain an underscore which could
+potentially lead to inconsistencies when running the `lxd recover` command.
+
+Volumes for virtual machines and ISO files require another two characters.
+The VMs root volume contains the `.b` block suffix whilst an ISO file ends on `.i`.
+
+In order to have enough space for the snapshot identifier, at least two more characters need to be
+reserved. You can try keeping the snapshots name at a minimum by setting [`snapshots.pattern`](storage-powerflex-snapshot-pattern)
+to `%d`. This will result in the following volume names:
+
+* Snapshot `snap0` for a virtual machine `v1`: `v_project1_v1@0`
+* Snapshot `snap1` for a custom volume `vol`: `u_project1_vol@1`
+
+(storage-powerflex-limitations)=
+### Limitations
+
+The `powerflex` driver has the following limitations:
+
+Limit of snapshots in a single vTree
+: From any volume in PowerFlex only 126 snapshots can be taken due to an internal limitation in the PowerFlex vTree.
+  This limit also applies to any child of any of the parent volumes snapshots.
```

Comment:
```suggestion
  This limit also applies to any child of any of the parent volume's snapshots.
```
I assume there's only one parent volume? Otherwise it might be `parent volumes'`

Reply:
There can be a chain of snapshots like (the entire graph is the vTree): 

```
parent -> snap1 -> snap5 -> snap8
       -> snap2            
       -> snap3 -> snap6
                -> snap7
```

But effectively there is only one parent so `volume's` seems to be the right choice.